fc_karpaty/src/test/java/tytarchuk/FCKarpatyTest.java

```diff
@@ -0,0 +1,31 @@
+package tytarchuk;
+
+import com.codeborne.selenide.Selenide;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class FCKarpatyTest extends ChromeSettings {
+    @BeforeTest
+    protected void setUpBrowser(){
+        super.setUpBrowser();
+    }
+
+    @Test
+    public void getResultsOfTeam() {
+        Selenide.open("https://fckarpaty.com");
```

Comment:
Page must be opened within page object class

Reply:
done